app/concepts/reservations/use_cases/update_reservation.rb

```diff
@@ -0,0 +1,34 @@
+module Reservations
+  module UseCases
+    class UpdateReservation
+      def initialize(reservation, status)
+        @reservation = reservation
+        @status = status
+      end
+
+      def call
+        confirm_reservation if status == 'confirmed'
+        cancel_reservation if status == 'cancelled'
```

Comment:
you have constants so you can use them here as well :D
```suggestion
        confirm_reservation if status == Reservation::CONFIRMED
        cancel_reservation if status == Reservation::CANCELLED
```
Now when you change value of the constant you don't have to update all places you are using `confirmed`